heroku2elk/heroku2Logstash.py

```diff
@@ -216,11 +228,20 @@ def configure_logger():
         logger.info("Start H2L in multi-processing mode")
         server = HTTPServer(app)
         server.bind(8080)
-        server.start(0)  # autodetect number of cores and fork a process for each
+        # autodetect number of cores and fork a process for each
+        server.start(0)
     else:
         logger.info("Start H2L in single-processing mode")
         app.listen(8080)
 
-    # instantiate an AMQP connection at start to create the queues (needed when logstash starts)
-    tornado.ioloop.IOLoop.instance().add_future(AMQPConnectionSingleton().get_channel(tornado.ioloop.IOLoop.instance()), lambda x: logger.info("pid:{} AMQP is connected".format(os.getpid())))
-    tornado.ioloop.IOLoop.instance().start()
+    # instantiate an AMQP connection at start to create the queues
+    # (needed when logstash starts)
+    ins = tornado.ioloop.IOLoop.instance()
+    ins.add_future(AMQPConnectionSingleton().get_channel(ins),
+                   lambda x: logger.info("AMQP is connected"))
+    if MainConfig.tornado_debug:
+        from dump_mem import record_top, start
```

Comment:
This code is not authorized in Python either

Reply:
Why ?